base_layer/wallet/src/transaction_service/storage/sqlite_db.rs

```diff
@@ -2037,9 +2056,13 @@ impl UnconfirmedTransactionInfoSql {
                 completed_transactions::coinbase_block_height,
             ))
             .filter(
-                completed_transactions::mined_height
-                    .is_null()
-                    .or(completed_transactions::status.eq(TransactionStatus::MinedUnconfirmed as i32)),
+                completed_transactions::status
+                    .ne(TransactionStatus::Imported as i32)
```

Comment:
Why are we excluding imported here?

Reply:
A coinbase is never imported maybe?

Reply:
because an import transaction can never be "unconfirmed" it is either pending on confirmed.

Reply:
For a bit more clarity this index method is used to pull transaction that need to be queried on a base node which imported transactions should not be.